src/GraphQL.Tests/Execution/AstFromValueTests.cs

```diff
@@ -96,7 +96,7 @@ public void converts_byte_to_byte_value()
             byte value = 12;
             var result = value.AstFromValue(schema, new ByteGraphType());
             result.ShouldNotBeNull();
-            result.ShouldBeOfType<ByteValue>();
+            result.ShouldBeOfType<GraphQL.Language.AST.ByteValue>();
```

Comment:
Is it definitely necessary?

Reply:
The other option would be to remove ByteValue and ByteGraphType from this file. This test is currently failing when running for me due to it expecting the GraphQL.Tests.Execution.ByteValue but is getting GraphQL.Language.AST.ByteValue.

Reply:
I think I get it. You added the built-in conversion and now the control does not fall into the test converter. The fact is that these tests were written before adding many primitive types (byte, sbyte, short, ushort, etc.) to the list of standard scalar types. Therefore, you do not need to change the namespace, but simply delete all unnecessary types from tests: `ByteValueConverter`, `ByteValue`, `ByteGraphType`.

Reply:
I kept `ByteValueConverter` due to it being used in `registers_ast_from_value_converters` so that it can still be tested that the `schema.RegisterValueConverter` method is still covered.